pmm-app/tests/databaseChecks_test.js

```diff
@@ -53,11 +55,25 @@ Scenario(
     }
 );
 
+Scenario(
+    'PMM-T338 Verify user can see correct colors in Failed Checks Panel at Home page @not-pr-pipeline',
```

Comment:
Hard-coding & verifying the colors is arguably the best thing to do in a test scenario.

Reply:
@atymchuk i did not get your point? what do you mean by this?